index.bs

```diff
@@ -606,6 +619,9 @@ When the {{getSettings()}} method is invoked on a video stream track, the user a
   <dt><dfn dict-member for="MediaTrackSettings"><code>exposureCompensation</code></dfn></dt>
   <dd>Current <a>exposure compensation</a> setting.  A value of 0 EV is interpreted as no exposure compensation.</dd>
 
+  <dt><dfn dict-member for="MediaTrackSettings"><code>exposureTime</code></dfn></dt>
+  <dd>Current <a>exposure time</a> setting. This field is only significant if {{MediaTrackSettings/exposureMode}} is {{manual}}.</dd>
```

Comment:
Correspondingly, l.620 should say it's only significant if `{{MediaTrackSettings/exposureMode}}` is not `{{manual}}` ?